src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImageFunctionalTest.groovy

```diff
@@ -0,0 +1,87 @@
+package com.bmuschko.gradle.docker.tasks.image
+
+import com.bmuschko.gradle.docker.AbstractFunctionalTest
+import com.bmuschko.gradle.docker.TestPrecondition
+import org.gradle.testkit.runner.BuildResult
+import spock.lang.Requires
+
+import static org.gradle.testkit.runner.TaskOutcome.SKIPPED
+import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
+
+@Requires({ TestPrecondition.DOCKER_SERVER_INFO_URL_REACHABLE })
+class DockerRemoveImageFunctionalTest extends AbstractFunctionalTest {
+
+    def "can remove image"() {
+        buildFile << """
+            import com.bmuschko.gradle.docker.tasks.image.Dockerfile
+            import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+            import com.bmuschko.gradle.docker.tasks.image.DockerRemoveImage
+
+            task dockerfile(type: Dockerfile) {
+                from 'ubuntu:12.04'
+            }
+
+            task buildImage(type: DockerBuildImage) {
+                dependsOn dockerfile
+                inputDir = file("build/docker")
+            }
+            
+            task removeImage(type: DockerRemoveImage) {
+                dependsOn buildImage
+                targetImageId { buildImage.getImageId() }
+            }
+        """
+
+        when:
+        BuildResult result = build('removeImage')
+
+        then:
+        result.output.contains("Removing image with ID")
+    }
+
+    def "can remove image tagged in multiple repositories"() {
+        buildFile << """
+            import com.bmuschko.gradle.docker.tasks.image.Dockerfile
+            import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+            import com.bmuschko.gradle.docker.tasks.image.DockerRemoveImage
+            import com.bmuschko.gradle.docker.tasks.image.DockerTagImage
+
+            task dockerfile(type: Dockerfile) {
+                from 'ubuntu:12.04'
+            }
+
+            task buildImage(type: DockerBuildImage) {
+                dependsOn dockerfile
+                inputDir = file("build/docker")
+            }
+            
+            task tagImage(type: DockerTagImage) {
+                dependsOn buildImage
+                repository = "repository"
+                tag = "tag2"
+                targetImageId { buildImage.getImageId() }
+            }
+            
+            task tagImageSecondTime(type: DockerTagImage) {
+                dependsOn tagImage
+                repository = "repository"
+                tag = "tag2"
+                targetImageId { buildImage.getImageId() }
+            }
+            
+            task removeImage(type: DockerRemoveImage) {
+                dependsOn tagImageSecondTime
+                force = true
+                targetImageId { buildImage.getImageId() }
+            }
+        """
+
+        when:
+        BuildResult result = build('removeImage')
+
+        then:
+        result.output.contains("Removing image with ID")
```

Comment:
@cdancy @orzeh 
I tried with virtualbox 5.1, 5.0, 4.3 and still same issue. I have also added virtualbox folder to my path, it didn't help.
Vagrant on his own does work well.

Here is the full stacktrace:

> org.gradle.api.GradleException: VirtualBox is not functional. Please check!
>         at com.bmuschko.gradle.vagrant.VagrantBasePlugin$_validateVagrantInstallation_closure3.doCall(VagrantBasePlugin.groovy:76)
>         at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
>         at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
>         at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:44)
>         at org.gradle.internal.event.BroadcastDispatch.dispatch(BroadcastDispatch.java:79)
>         at org.gradle.internal.event.BroadcastDispatch.dispatch(BroadcastDispatch.java:30)
>         at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
>         at com.sun.proxy.$Proxy52.graphPopulated(Unknown Source)
>         at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter.execute(DefaultTaskGraphExecuter.java:108)
>         at org.gradle.execution.SelectedTaskExecutionAction.execute(SelectedTaskExecutionAction.java:37)
>         at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:37)
>         at org.gradle.execution.DefaultBuildExecuter.access$000(DefaultBuildExecuter.java:23)
>         at org.gradle.execution.DefaultBuildExecuter$1.proceed(DefaultBuildExecuter.java:43)
>         at org.gradle.execution.DryRunBuildExecutionAction.execute(DryRunBuildExecutionAction.java:32)
>         at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:37)
>         at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:30)
>         at org.gradle.initialization.DefaultGradleLauncher$4.run(DefaultGradleLauncher.java:153)
>         at org.gradle.internal.Factories$1.create(Factories.java:22)
>         at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
>         at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:53)
>         at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:150)
>         at org.gradle.initialization.DefaultGradleLauncher.access$200(DefaultGradleLauncher.java:32)
>         at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:98)
>         at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:92)
>         at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
>         at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:63)
>         at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:92)
>         at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:83)
>         at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildController.run(InProcessBuildActionExecuter.java:99)
>         at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
>         at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
>         at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:48)
>         at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:30)
>         at org.gradle.launcher.exec.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:81)
>         at org.gradle.launcher.exec.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:46)
>         at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:51)
>         at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:28)
>         at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:43)
>         at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:173)
>         at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:239)
>         at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:212)
>         at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:35)
>         at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
>         at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
>         at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
>         at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:205)
>         at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:169)
>         at org.gradle.launcher.Main.doAction(Main.java:33)
>         at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
>         at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:55)
>         at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:36)
>         at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
>         at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:30)
>         at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:127)
>         at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:56)
>